apps/ui/src/views/Proposal.vue

```diff
@@ -156,8 +156,12 @@ watchEffect(() => {
             <div class="pt-2">
               <UiLoading v-if="votingPowerStatus === 'loading'" />
               <button v-else class="text-skin-link text-lg" @click="props.onClick">
-                <IH-exclamation v-if="votingPowerStatus === 'error'" class="inline-block mr-1" />
-                {{ props.formattedVotingPower }}
+                <IH-lightning-bolt class="inline-block" />
+                <IH-exclamation
+                  v-if="votingPowerStatus === 'error'"
+                  class="inline-block ml-1 text-rose-500"
+                />
+                <span v-else class="ml-2" v-text="props.formattedVotingPower" />
```

Comment:
This doesn't look aligned:
<img width="359" alt="image" src="https://github.com/snapshot-labs/sx-monorepo/assets/1968722/0715e898-5ebb-4528-9457-2c8850bbc20b">

With align items center
<img width="347" alt="image" src="https://github.com/snapshot-labs/sx-monorepo/assets/1968722/272c87f7-85bc-40ed-8dd4-7accab16feff">